packages/tailwind/src/tailwind.tsx

```diff
@@ -90,8 +90,15 @@ function processElement(
   );
 
   // if this is a component, then we render it and recurse it through processElement
-  if (typeof modifiedElement.type === "function") {
-    const component = modifiedElement.type as React.FC;
+  if (
+    typeof modifiedElement.type === "function" ||
+    // @ts-expect-error - we know this is a component that may have a render function
```

Comment:
I'm uncomfortable with this ts-expect-error, but struggled with what type to otherwise declare for the `element` prop on line 22 that would cover both React.Element and a forwarded ref element. :(